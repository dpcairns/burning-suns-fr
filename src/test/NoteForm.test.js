import React from 'react';
import { shallow } from 'enzyme';
import Test from '../NoteForm';


describe('First React component test with Enzyme', () => {
    it('renders without crashing', () => {
      const wrapper = shallow(<Test />);
      expect(wrapper).toMatchSnapshot()
     });
 });