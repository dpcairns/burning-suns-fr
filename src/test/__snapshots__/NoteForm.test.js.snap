// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First React component test with Enzyme renders without crashing 1`] = `
<div>
  <WithStyles(ForwardRef(Container))
    component="main"
    maxWidth="xs"
  >
    <div
      className="makeStyles-paper-1"
    >
      <WithStyles(ForwardRef(Typography))
        gutterBottom={true}
        variant="h6"
      >
        Add a 
        Note
      </WithStyles(ForwardRef(Typography))>
      <form
        className="makeStyles-form-2"
        onSubmit={[Function]}
      >
        <WithStyles(ForwardRef(Grid))
          container={true}
          spacing={2}
        >
          <WithStyles(ForwardRef(Grid))
            item={true}
            xs={12}
          >
            <WithStyles(ForwardRef(TextField))
              fullWidth={true}
              id="outlined-title"
              label="Title"
              onChange={[Function]}
              value=""
              variant="outlined"
            />
          </WithStyles(ForwardRef(Grid))>
          <WithStyles(ForwardRef(Grid))
            item={true}
            sm={6}
            xs={12}
          >
            <WithStyles(ForwardRef(TextField))
              helperText="Please select from your starred cities"
              id="standard-select-city"
              label="Select"
              onChange={[Function]}
              select={true}
              value=""
            >
              <WithStyles(ForwardRef(MenuItem))
                key="undefined, undefined"
                value=""
              >
                undefined , undefined
              </WithStyles(ForwardRef(MenuItem))>
            </WithStyles(ForwardRef(TextField))>
          </WithStyles(ForwardRef(Grid))>
          <WithStyles(ForwardRef(Grid))
            item={true}
            sm={6}
            xs={12}
          >
            <WithStyles(ForwardRef(FormControlLabel))
              control={
                <ForwardRef(WithStyles)
                  checked={false}
                  color="secondary"
                  name="saveCard"
                  onChange={[Function]}
                />
              }
              label="Save as a wish"
            />
          </WithStyles(ForwardRef(Grid))>
          <WithStyles(ForwardRef(Grid))
            item={true}
            xs={12}
          >
            <DatePicker
              calendarIcon={
                <svg
                  className="react-date-picker__calendar-button__icon react-date-picker__button__icon"
                  height={19}
                  stroke="black"
                  strokeWidth={2}
                  viewBox="0 0 19 19"
                  width={19}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <rect
                    fill="none"
                    height="15"
                    width="15"
                    x="2"
                    y="2"
                  />
                  <line
                    x1="6"
                    x2="6"
                    y1="0"
                    y2="4"
                  />
                  <line
                    x1="13"
                    x2="13"
                    y1="0"
                    y2="4"
                  />
                </svg>
              }
              clearIcon={
                <svg
                  className="react-date-picker__clear-button__icon react-date-picker__button__icon"
                  height={19}
                  stroke="black"
                  strokeWidth={2}
                  viewBox="0 0 19 19"
                  width={19}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <line
                    x1="4"
                    x2="15"
                    y1="4"
                    y2="15"
                  />
                  <line
                    x1="15"
                    x2="4"
                    y1="4"
                    y2="15"
                  />
                </svg>
              }
              closeCalendar={true}
              isOpen={null}
              onChange={[Function]}
              returnValue="start"
              value={2020-05-22T16:24:04.168Z}
            />
          </WithStyles(ForwardRef(Grid))>
          <WithStyles(ForwardRef(Grid))
            item={true}
            xs={12}
          >
            <WithStyles(ForwardRef(TextField))
              fullWidth={true}
              helperText=""
              id="outlined-textarea"
              label="Message"
              multiline={true}
              onChange={[Function]}
              placeholder="Add text here"
              value=""
              variant="outlined"
            />
          </WithStyles(ForwardRef(Grid))>
          <WithStyles(ForwardRef(Grid))
            item={true}
            xs={12}
          >
            <WithStyles(ForwardRef(Button))
              color="primary"
              fullWidth={true}
              type="submit"
              variant="contained"
            >
              Add 
              Note
            </WithStyles(ForwardRef(Button))>
          </WithStyles(ForwardRef(Grid))>
        </WithStyles(ForwardRef(Grid))>
      </form>
    </div>
  </WithStyles(ForwardRef(Container))>
</div>
`;
